packages/lodestar/src/network/nodejs/util.ts

```diff
@@ -66,6 +65,12 @@ export async function createNodeJsLibp2p(
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const {NodejsNode} = await Function("return import('./bundle.mjs')")();
```

Comment:
Write a nice long comment explaining this hahaha

Reply:
maybe you can import the type with `import type` just to make Typescript happy and not have everything as any

Reply:
lmao the most malicious looking code ever. Hopefully we finish #3978 instead of this.